Makefile

```diff
@@ -664,6 +664,9 @@ cpplint:
 pylint:
 	python3 -m pylint --rcfile=$(ROOTDIR)/ci/other/pylintrc --ignore-patterns=".*\.so$$,.*\.dll$$,.*\.dylib$$" python/mxnet tools/caffe_converter/*.py
 
+sample_lib:
+	$(CXX) -shared -fPIC -std=c++11 example/lib_ops/gemm_lib.cc -o libsample_lib.so -I include/mxnet
+
```

Comment:
Can we please add an rm libsample_lib.so to the clean target so this is removed and the project can be cleaned up properly?

Reply:
that's a good point